extensions-core/google-extensions/src/test/java/org/apache/druid/storage/google/GoogleStorageTest.java

```diff
@@ -159,7 +160,33 @@ public void testGetMetadata() throws IOException
     EasyMock.replay(mockStorage, blob);
 
     GoogleStorageObjectMetadata objectMetadata = googleStorage.getMetadata(BUCKET, PATH);
-    assertEquals(objectMetadata, new GoogleStorageObjectMetadata(BUCKET, PATH, SIZE, UPDATE_TIME.toEpochSecond()));
+    assertEquals(
+        objectMetadata,
+        new GoogleStorageObjectMetadata(BUCKET, PATH, SIZE, UPDATE_TIME.toEpochSecond() * 1000)
+    );
+
+  }
+  @Test
+  public void testGetMetadataMismatch() throws IOException
```

Comment:
what is this test for?

Reply:
Oh, I think it is the same as the above test `testGetMetadataMismatch()` but verifies that the timestamp is __not in seconds__ .

@gargvishesh , I think the previous test `testGetMetadata()` is enough.